app/Http/Controllers/CommentController.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Comment;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Exception;
+
+
+class CommentController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     */
+    public function index()
+    {
+        return Comment::all();
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     */
+    public function create()
+    {
+        //
+    }
+
+
+    public function store(Request $request)
+    {
+        try{
+            $validated_data = $request->validate([
+                'comment' => 'required|string',
+                
+            ]);
+            Comment::create($validated_data);
+            return response()->json([
+                'message' => 'Comment added successfully'
+            ], 201);
+        } catch (Exception $e){
+            return response()->json(['message' => 'error', 'error' => $e->getMessage()], 500);
+        }
+    }
+    
+    public function show($id)
+    {
+        $comment = Comment::find($id);
+        if (!$comment){
+            return response()->json(['message' => 'Comment not found'], 404);
+        }
+        return response()->json(['comment' => $comment], 200);
+    }
+
+
+    public function edit(string $id)
+    {
+        // Get the comment by its id and pass it to the view
+        // return View('comments.edit')->withComment(Comment::find($id));
```

Comment:
This method should get a given comment and send it to the user. You can implicitly bind the comment at the Route level so it becomes a simple return  like

```php
use App\Models\Comment

public function edit(Comment $comment)
{
    // Other checks to come in later
    return $comment;
}

Reply:
Done